.env.example

```diff
@@ -31,3 +31,6 @@ OPERATOR_API_KEY=a-secret-that-should-get-changed
 STRIPE_API_KEY=get-this-from-your-stripe-account
 
 NEIGHBORHOOD_TIME_ZONE="Pacific Time (US & Canada)"
+
+# Configure Square environment
+# SQUARE_ENV="production"|"sandbox"
```

Comment:
This probably should be set in the Marketplace for now, rather than a global variable.